systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -456,6 +457,61 @@ void testDynamicallySetBridgeLoggingLevels() throws InterruptedException {
         assertThat("Bridge pod should not roll", DeploymentUtils.depSnapshot(KafkaBridgeResources.deploymentName(CLUSTER_NAME)), equalTo(bridgeSnapshot));
     }
 
+    @Test
+    void testDynamicallySetClusterOperatorLoggingLevels() {
+        Map<String, String> coPod = DeploymentUtils.depSnapshot(STRIMZI_DEPLOYMENT_NAME);
+        String coPodName = kubeClient().listPodsByPrefixInName(STRIMZI_DEPLOYMENT_NAME).get(0).getMetadata().getName();
+
+        String log4jConfig =
+            "name = COConfig\n" +
+            "monitorInterval = 30\n" +
+            "\n" +
+            "    appender.console.type = Console\n" +
+            "    appender.console.name = STDOUT\n" +
+            "    appender.console.layout.type = PatternLayout\n" +
+            "    appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n\n" +
+            "\n" +
+            "    rootLogger.level = INFO\n" +
+            "    rootLogger.appenderRefs = stdout\n" +
+            "    rootLogger.appenderRef.console.ref = STDOUT\n" +
+            "    rootLogger.additivity = false\n" +
+            "\n" +
+            "    # Kafka AdminClient logging is a bit noisy at INFO level\n" +
+            "    logger.kafka.name = org.apache.kafka\n" +
+            "    logger.kafka.level = WARN\n" +
+            "    logger.kafka.additivity = false\n" +
+            "\n" +
+            "    # Zookeeper is very verbose even on INFO level -> We set it to WARN by default\n" +
+            "    logger.zookeepertrustmanager.name = org.apache.zookeeper\n" +
+            "    logger.zookeepertrustmanager.level = WARN\n" +
+            "    logger.zookeepertrustmanager.additivity = false";
```

Comment:
How does this one differ from the original one? I didn't compare it in detail. But it looks similar.

Reply:
You are right, I'm gonna change it -> the only difference was in lining and white spaces. Thanks!